src/System.Diagnostics.TraceSource/tests/TraceClassTests.cs

```diff
@@ -341,9 +351,42 @@ public void TraceTest01()
         }
 
         [Fact]
```

Comment:
Just FYI: we can't use AppContext.TargetFrameworkName yet as it is not available in .NET 46 so we would break the desktop tests in runtime.